src/assets/stylesheets/components/content/_reference.scss

```diff
@@ -1,5 +1,16 @@
 .reference-container {
   display: flex;
-  justify-content: flex-end;
   margin-bottom: $minPadding;
+
+  &-left {
+    justify-content: flex-start;
+  }
+
+  &-right {
+    justify-content: flex-end;
+  }
+
+  &-center {
+    justify-content: center;
+  }
```

Comment:
added positioning classes based on the `position` property in the reference modal JSON